packages/terra-tabs/src/common-tabs/Tab.module.scss

```diff
@@ -153,9 +164,29 @@
     }
   }
 
+  .tabs-remove-button {
+    background-color: var(--terra-tabs-close-tab-background-color, transparent);
+    border: 0;
+    border-radius: 0.5714rem;
+    color: var(--terra-tabs-remove-button-color, #595f62);
+    cursor: pointer;
+    display: inline-block;
+    font-family: inherit;
+    font-size: 1rem;
+    height: 1rem;
+    line-height: 1;
+    margin: 0;
+    margin-left: 10px;
+    min-width: 1.02857rem;
+    outline: none;
+    padding: 0;
+    right: 0;
+  }
+
   [data-tab-is-calculating='true'] {
     > .tab {
       flex: 0 0 auto;
     }
   }
+
```

Comment:
```suggestion
```
nit : empty lines should be removed.

Reply:
Done https://github.com/cerner/terra-framework/commit/fcb1925802030dfb5be6bf821a0a978a5c82d40c